Components/Button/button.css

```diff
@@ -0,0 +1,91 @@
+.button-container {
+  background-color: var(--lightGray-color);
+  min-height: 20rem;
+  border: 1rem solid var(--lightGray-color);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 2rem;
+  flex-wrap: wrap;
+}
+
+.btn {
+  border: none;
+  padding: 1rem;
+  text-align: center;
+  min-width: 10rem;
+  cursor: pointer;
+  font-weight: 600;
+  color: var(--white-color);
+  background-color: var(--primary-color);
+  border-radius: 0.3rem;
+}
+
+.btn-primary {
+  background-color: var(--primary-color);
+}
+
+.btn-secondary {
+  background-color: var(--secondary-color);
+}
+
+.btn-error {
+  background-color: var(--danger-color);
+}
+
+.btn-success {
+  background-color: var(--success-color);
+}
+
+.outline {
+  background-color: var(--white-color);
+  color: var(--primary-color);
+  border: 1px solid var(--primary-color);
+}
+
+.outline:hover {
+  background-color: var(--primary-color);
+  color: var(--white-color);
+  border: none;
+}
+
+.outline-secondary {
+  background-color: var(--white-color);
+  color: var(--secondary-color);
+  border: 1px solid var(--secondary-color);
+}
+
+.outline-secondary:hover {
+  background-color: var(--secondary-color);
+  color: var(--white-color);
+  border: none;
+}
+
+.outline-error {
+  background-color: var(--white-color);
+  color: var(--danger-color);
+  border: 1px solid var(--danger-color);
+}
+
+.outline-error:hover {
+  background-color: var(--danger-color);
+  color: var(--white-color);
+  border: none;
+}
+
+.outline-success {
+  background-color: var(--white-color);
+  color: var(--success-color);
+  border: 1px solid var(--success-color);
+}
+
+.outline-success:hover {
+  background-color: var(--success-color);
+  color: var(--white-color);
+  border: none;
+}
+
+.btn-icon {
+  position: absolute;
+  right: 64rem;
```

Comment:
icon overlapping the text
![image](https://user-images.githubusercontent.com/87164978/152967206-8555564d-ccc2-447e-b3e0-43e169d61e34.png)

Reply:
+1

Reply:
Issue resolved .Thanks for review